contrib/platform/src/com/sun/jna/platform/win32/WinDef.java

```diff
@@ -1774,4 +1774,54 @@ public HGLRCByReference(HGLRC h) {
             super(h);
         }
     }
+
+    /**
+     * COORD structure
+     */
+    @FieldOrder({ "X", "Y" })
+    public static class COORD extends Structure {
+
+        public short X;
+        public short Y;
+
+        @Override
+        public String toString() {
+            return String.format("COORD[%s,%s]", X, Y);
+        }
+    }
+
+    /**
+     * SMALL_RECT structure
+     */
+    @FieldOrder({ "Left", "Top", "Right", "Bottom" })
+    public static class SMALL_RECT extends Structure {
+
+        public short Left;
+        public short Top;
+        public short Right;
+        public short Bottom;
+
+        @Override
+        public String toString() {
+            return String.format("SMALL_RECT[%s,%s,%s,%s]", Left, Top, Right, Bottom);
```

Comment:
Same comment here on `%d` vs `%s`. Also just my opinion, but while (X,Y) is standard for coordinates, I'm not sure there's a generally accepted standard order of values for a "rectangle".  I've seen (left,top,width,height) for example, so this would be more clear with `L=%d,T=%d,R=%d,B=%d` or possibly combining left/top and right/bottom as a coordinate pair `(%d,%d)(%d,%d)`.

Reply:
I just added `toString()` for the sake of completeness. I don't actually use them in my code. I'm happy to make any changes you see fit.

Reply:
I'm just offering suggestions!

Reply:
Sure thing. I switched to `()` because the other classes use that as well.